packages/paste-style-props/src/types/border.ts

```diff
@@ -58,11 +59,19 @@ export interface BorderStyleProps {
   borderLeftStyle?: BorderLeftStyle;
 }
 
+export type BorderCollapseProperty = Properties['borderCollapse'];
+export type BorderCollapse = ResponsiveValue<BorderCollapseProperty>;
+
+export type BorderSpacingProperty = Properties['borderSpacing'];
+export type BorderSpacing = ResponsiveValue<BorderSpacingProperty>;
+
```

Comment:
Pulled in border props from generic css property types to keep border style props all in one place